src/overview/components/ResultList.jsx

```diff
@@ -27,6 +27,15 @@ const timeGapToSpaceGap = makeNonlinearTransform({
 })
 
 const ResultList = ({searchResult}) => {
+
+    //if no result found display relavant message
+    if(searchResult.rows.length === 0){
+        return <p className={styles.message}>
```

Comment:
unless is a project decision what do you think of the style (I've seen it most often because it keeps opening close lines)
```
return (
    <p className={styles.message}>
        No result found
    </p>
)